### PR TITLE
doc: Doc fixes for 2.9

### PR DESCRIPTION
**Pull request by umapraseeda**
Doc check and fixes for 2.9
NCSDK-30694

### CODE REVIEW

File: samples/cellular/location/README.rst [hunk 1]
```diff
@@ -84,7 +84,7 @@ See :ref:`cmake_options` for more instructions on how to add these options.
 
 .. note::
    The Thingy:91 X build supports Wi-Fi by default.
-	You do not need to add any overlays.
+  	You need not add any overlays.
```
Comment (peknis) [inline samples/cellular/location/README.rst:87]: Keep the text as it, but align with the line above.

File: samples/suit/recovery/README.rst [hunk 2]
```diff
@@ -51,8 +51,8 @@ To achieve this, the appropriate devicetree overlay files from the main applicat
 
 To do this, add the :file:`recovery.overlay` and :file:`recovery_hci_ipc.ovelay` files in the main application's :ref:`configuration_system_overview_sysbuild` directory.
 The former file will be passed automatically to the recovery application image and the latter to the recovery radio image.
-These devicetree files must define the ``cpuapp_recovery_partition`` and ``cpurad_recovery_partition`` nodes respectively.`
-For an example, see the files in the ``samples/suit/smp_transfer`` sample.
+These devicetree files must define the ``cpuapp_recovery_partition`` and ``cpurad_recovery_partition`` nodes respectively.
+For an example, see the files in the :file:`samples/suit/smp_transfer` sample.
```
Comment (peknis) [inline samples/suit/recovery/README.rst:55]:
```suggestion
For an example, see the files in the :file:`samples/suit/smp_transfer` folder.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -318,7 +318,7 @@ The following table indicates the software maturity levels of the support for ea
      - --
      - --
      - --
-   * - **Wi-Fi**
+   * - **Wi-Fi®**
```
Comment (richabp) [inline doc/nrf/releases_and_maturity/software_maturity.rst:321]: I think it should be added to Bluetooth® in line 193.

File: doc/nrf/test_and_optimize/optimizing/power_general.rst
Comment (richabp) [inline doc/nrf/test_and_optimize/optimizing/power_general.rst:1]: Bluetooth® Mesh at line 179?

File: samples/matter/template/README.rst
Comment (richabp) [inline samples/matter/template/README.rst:1]: Can remove ® from Line 52, 142.

**Review by richabp — CHANGES_REQUESTED** [review]

File: applications/matter_bridge/README.rst
[the remark this answers is not in the document]
Comment (umapraseeda) [reply applications/matter_bridge/README.rst:1]: Fixed

**Comment by umapraseeda** [comment]
@nrfconnect/ncs-protocols-serialization please check and approve.

**Review by richabp — APPROVED** [review]

File: doc/_utils/redirects.py [hunk 4]
```diff
@@ -207,6 +207,7 @@
     ("device_guides/working_with_nrf/nrf54l/testing_dfu", "app_dev/device_guides/nrf54l/fota_update"), # Testing the DFU solution (removed after 2.8.0)
     ("app_dev/device_guides/working_with_nrf/nrf54l/testing_dfu", "app_dev/device_guides/nrf54l/fota_update"),
     ("app_dev/device_guides/nrf54l/testing_dfu", "app_dev/device_guides/nrf54l/fota_update"),
+    ("app_dev/device_guides/nrf54l/snippets", "app_dev/device_guides/nrf54l/index"), # Removed after 2.8.0
```
Comment (greg-fer) [inline doc/_utils/redirects.py:210]: If possible, mention the page title in the comment, to match other comment entries.

**Review by greg-fer — APPROVED** [review]
One nit.

**Comment by shanthanordic** [comment]
is this must go in for 2.9 ? @umapraseeda

**Review by Damian-Nordic — APPROVED** [review]

**Pull request merged**